packages/manager/apps/sap-features-hub/src/utils/summaryExport.ts

```diff
@@ -1,21 +1,28 @@
 import { InstallationFormValues } from '@/types/form.type';
 
-const getSummaryJSON = (form: InstallationFormValues) => {
+export const isEmptyObject = (object: Record<string, any> | undefined) => {
+  const isEmpty = !object || Object.values(object).every((val) => !val);
+  return isEmpty;
+};
+
+export const getSummaryJSON = (form: InstallationFormValues) => {
   return JSON.stringify({
     applicationServers: {}, // step not developed yet
     applicationType: form.applicationType,
     applicationVersion: form.applicationVersion,
-    bucketBackint: {
-      accessKey: form.bucketBackint?.accessKey,
-      endpoint: form.bucketBackint?.endpoint,
-      id: form.bucketBackint?.id,
-      secretKey: '',
-    },
+    bucketBackint: isEmptyObject(form.bucketBackint)
+      ? undefined
+      : {
+          accessKey: form.bucketBackint?.accessKey,
+          endpoint: form.bucketBackint?.endpoint,
+          id: form.bucketBackint?.id,
+          secretKey: form.bucketBackint?.secretKey,
```

Comment:
```suggestion
          ...form.bucketBackint
```

Not better if you want duplicate all value ?